src/certificateList.jsx

```diff
@@ -90,44 +93,81 @@ function getCAName(cas, cert) {
+                        </label>
+                        <span id={`${idPrefix}-general-validity`}>
+                            {prettyTime(cert["not-valid-before"].v)
+                            +  _(" to ") + prettyTime(cert["not-valid-after"].v)}
+                        </span>
+                    </>}
+                    {cert.autorenew && cert.autorenew.v && <>
+                        <label className='control-label label-title' htmlFor={`${idPrefix}-general-autorenewal`}>
+                            {_("Auto-renewal")}
+                        </label>
+                        <span id={`${idPrefix}-general-autorenewal`}>{cert.autorenew.v ? _("Yes") : _("No")}</span>
+                    </>}
+                    {cert.stuck && cert.stuck.v && <>
+                        <label className='control-label label-title' htmlFor={`${idPrefix}-general-stuck`}>{_("Stuck")}</label>
+                        <span id={`${idPrefix}-general-stuck`}>{cert.stuck.v ? _("Yes") : _("No")}</span>
+                    </>}
+                </div>
+            </Flex>
         </Flex>
-        <Flex breakpointMods={[{modifier: FlexModifiers["column", "flex-1"]}]}>
-            <div className="ct-form">
-                {cert["not-valid-after"] && cert["not-valid-after"].v !== 0 && <>
-                    <label className='control-label label-title' htmlFor={`${idPrefix}-general-validity`}>
-                        {_("Valid")}
-                    </label>
-                    <span id={`${idPrefix}-general-validity`}>
-                        {prettyTime(cert["not-valid-before"].v)
-                        +  _(" to ") + prettyTime(cert["not-valid-after"].v)}
-                    </span>
-                </>}
-                {cert.autorenew && cert.autorenew.v && <>
-                    <label className='control-label label-title' htmlFor={`${idPrefix}-general-autorenewal`}>
-                        {_("Auto-renewal")}
-                    </label>
-                    <span id={`${idPrefix}-general-autorenewal`}>{cert.autorenew.v ? _("Yes") : _("No")}</span>
-                </>}
-                {cert.stuck && cert.stuck.v && <>
-                    <label className='control-label label-title' htmlFor={`${idPrefix}-general-stuck`}>{_("Stuck")}</label>
-                    <span id={`${idPrefix}-general-stuck`}>{cert.stuck.v ? _("Yes") : _("No")}</span>
-                </>}
-            </div>
+
+        <Text component={TextVariants.h5}>
+            {_("Signing request properties")}
+        </Text>
+
+        <Flex breakpointMods={[{modifier: FlexModifiers["justify-content-space-between"]}]}>
+            <Flex breakpointMods={[{modifier: FlexModifiers["column", "flex-1"]}]}>
+                <div className='ct-form'>
+                    {cert.subject && cert.subject.v && <>
+                        <label className='control-label label-title' htmlFor={`${idPrefix}-general-subject`}>
+                            {_("Subject Name")}
```

Comment:
Patternfly4 (unlike Patternfly3) recommends Sentence style
https://www.patternfly.org/v4/design-guidelines/content/grammar-and-terminology#sentence-style

So this would become _Subject name_